examples/va/nfv/sriov/edpm/values.yaml

```diff
@@ -158,8 +85,70 @@ spec:
         # edpm firewall, change the allowed CIDR if needed
         edpm_sshd_configure_firewall: true
         edpm_sshd_allowed_ranges:
-          - 192.168.122.0/24 # CHANGEME
+          - 192.168.122.0/24
         # SELinux module
         edpm_selinux_mode: enforcing
         # SRIOV settings
-        edpm_neutron_sriov_agent_SRIOV_NIC_physical_device_mappings: 'sriov-phy4:eno4' # CHANGEME
+        edpm_neutron_sriov_agent_SRIOV_NIC_physical_device_mappings: 'sriov-phy4:eno4'
+    nodes:
+      edpm-compute-0:
+        ansible:
+          ansibleHost: 192.168.122.100
+        hostName: edpm-compute-0
+        networks:
+        - defaultRoute: true
+          fixedIP: 192.168.122.100
+          name: CtlPlane
+          subnetName: subnet1
+        - name: InternalApi
+          subnetName: subnet1
+        - name: Storage
+          subnetName: subnet1
+        - name: Tenant
+          subnetName: subnet1
+      edpm-compute-1:
+        ansible:
+          ansibleHost: 192.168.122.101
+        hostName: edpm-compute-1
+        networks:
+        - defaultRoute: true
+          fixedIP: 192.168.122.101
+          name: CtlPlane
+          subnetName: subnet1
+        - name: InternalApi
+          subnetName: subnet1
+        - name: Storage
+          subnetName: subnet1
+        - name: Tenant
+          subnetName: subnet1
+    services:
+    - bootstrap
+    - configure-network
+    - validate-network
+    - install-os
+    - configure-os
+    - run-os
+    - reboot-os
+    - libvirt
+    - ovn
+    - nova-custom-sriov
+    - neutron-sriov
+    - neutron-metadata
+  nova:
+    compute:
+      conf: |
+        [DEFAULT]
+        reserved_host_memory_mb = 4096
+        reserved_huge_pages = node:0,size:4,count:4096000
+        reserved_huge_pages = node:1,size:4,count:4096000
```

Comment:
i realised i failed at basic math

orgianly you were reserving 4G total for the host use but the example i provided are now resving 32G for the numa aware part

so i should have done the following
4096MB -> 4193280 KB
419328KB / 4KB pages = 1048320 pages
1048320 pages / 2 numa nodes = 524160 pages per numa node

so if you have 
reserved_host_memory_mb = 4096
and 2 numa nodes on the host you should have
reserved_huge_pages = node:0,size:4,count:524160
reserved_huge_pages = node:1,size:4,count:524160

i should really just modify nova to do this by default so humans don't have to do basic math

Reply:
Thank you for crunching the numbers.  I've adjusted them.